app/models/communities.rb

```diff
@@ -0,0 +1,59 @@
+class Communities
+  require "yaml"
+
+  @local_chapters = {}
+
+  def self.local_chapters(locale)
+    @local_chapters[locale] = local_chapter_for(locale)
+  end
+
+  class << self
```

Comment:
Why was `self.` good enough before but now you want to resort to black magic to define these class methods?